### ISSUE TITLE
File.deselect not working

### ISSUE THREAD

**Issue by Socratemus** [mafintosh/torrent-stream#181]
I know that there is no activity in this repository for some time now but has anyone encounted this issue?

**Comment by desche42** [comment]
Yes. I use file.select() to select video files and file.deselect() for others line txt or exe files. All are downloaded

**Comment by ikaio** [comment]
same problem

**Comment by gineff** [comment]
engine.selection.length = 0
